js/src/dapps/tokenreg/Tokens/Token/token.js

```diff
@@ -185,13 +276,14 @@ export default class Token extends Component {
   }
 
   onMetaLookup = () => {
-    let query = this.state.metaQuery;
+    let keyIndex = this.state.metaKeyIndex;
```

Comment:
Yes. Unless we need to modify things. The best approach - just use const, if you assign something and it complains, let it :)